Library/Homebrew/utils.rb

```diff
@@ -267,7 +292,11 @@ def curl(*args)
   safe_system curl, *args
 end
 
-def puts_columns(items, highlight = [])
+def output_length string
```

Comment:
I think a better place for this method would be the `Tty` class. Maybe also rename it to `plain_length`? And haven't we settled on always putting the arguments in parentheses on method definition?

One more thought: Maybe let the method return the string with ANSI escapes removed? It would be the responsibility of the caller to call `length` on the result, but that would make this function a lot more useful in case we ever want to get plain text from formatted strings. If we can agree on that, an even better name might be `strip_ansi`.